### ISSUE TITLE
EditableEnumProperty hide bottom part of character in edit status

### ISSUE THREAD

**Issue by mviz-mdc** [ros-visualization/rviz#1599]
There is a UI issue for EditableEnumProperty.  When it enter edit status, It will hide the bottom part of 'g', 'q', 'y' etc. It make a little confusing.
![image](https://user-images.githubusercontent.com/81354680/112446745-7ce0a600-8d8b-11eb-9c05-5a00814237cb.png)

If modify code here, it could be solve.
![image](https://user-images.githubusercontent.com/81354680/112446802-8ec24900-8d8b-11eb-8e91-dc01ef01653b.png)

Effect after modification:
![image](https://user-images.githubusercontent.com/81354680/112446594-56226f80-8d8b-11eb-9a39-264160c0223a.png)

If there is any better way to fix it, maybe we could have a discuss here.

### Environment
* OS Version: Ubuntu 18.04
* ROS Distro: Melodic
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
```
[ INFO] [1616568586.970028200]: rviz version 1.13.15
[ INFO] [1616568586.970076100]: compiled against Qt version 5.9.5
[ INFO] [1616568586.970083600]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1616568586.973661300]: Forcing OpenGl version 0.
[ INFO] [1616568587.067990800]: Stereo is NOT SUPPORTED
[ INFO] [1616568587.068078500]: OpenGl version: 3.1 (GLSL 1.4).
```

**Comment by rhaschke** [comment]
Thanks for this issue report and the proposed solution. Do you mind filing an appropriate pull request? Thanks.

**Comment by rhaschke** [comment]
Fix applied in b3ce39751df614999039359bf6cb646577cbbae3. Thanks again.
